lib/classes/CLI.js

```diff
@@ -128,6 +129,13 @@ class CLI {
   generateMainHelp() {
     this.consoleLog('');
 
+    if (isInChina) {
```

Comment:
@timqian Below we already have section in which we document `--help-components` I believe we should not show two of them. 

I think it'll be best if we hide below one at same time

Reply:
Just removed the duplicated info